webapp/components/wpt-results.html

```diff
@@ -482,26 +511,45 @@ <h3>
                 revision: 'diff',
                 browser_name: 'diff',
               };
+              this.fetchDiff();
             }
           })
         );
       }
 
       async fetchResults() {
```

Comment:
Using `async` here still looks like a bug. Even if `this.load` return a promise (not checked) it's not returned, so fetchResults() would look like `Promise.resolve(undefined)` with side-effects happening after the promise is settled.